ailment/expression.py

```diff
@@ -131,7 +131,6 @@ def __str__(self):
 
     def likes(self, other):
         return type(self) is type(other) and \
-            self.tmp_idx == other.tmp_idx and \
```

Comment:
I don't think this is correct. Are you saying `t0 == t1`? Note that this is not the `idx` that AIL manager assigns.